manifests/charts/istio-cni/templates/clusterrolebinding.yaml

```diff
@@ -17,7 +17,7 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ template "name" . }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ if .Values.namespaceOverride }}{{ .Values.namespaceOverride }}{{ else }}{{ .Release.Namespace }}{{ end }}
```

Comment:
Can we

1. just call it `.Values.namespace` 
2. Use `coalese` here to prefer `.Values.namespace` ELSE `.Release.Namespace`

Otherwise SGTM

Reply:
Done